accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -687,6 +687,26 @@ It's a compile-time error if:
     class M with S {} // Error, for several reasons.
     ```
 
+*   A mixin has a declared `on` type from another library which is marked
+    `final` _(with some exceptions for platform libraries)_.
+
+    More formally:
+    A `mixin` declaration *D* from library *L* has a declared `on` type *F*
+    marked `final` (so necessarily a `class` declaration) in a library *K*,
+    and neither
+    * *L* and *K* is the same library, nor
+    * *K* is a platform library and *L* is a pre-feature library.
+
+    ```dart
+    // a.dart
+    final class F {}
+
+    // b.dart
+    import 'a.dart';
+
+    mixin M on F {}  // Error.
+    ```
+
```

Comment:
In line 719 below: Mixins can't be `interface` or `final` any more, so "or mixin" isn't needed.

Reply:
Done.